src/components/common/Icon/Icon.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Icon = ({ icon, iconColor, size }) => (
-  <i className="material-icons" style={{ color: iconColor, fontSize: size }}>
+const Icon = ({ icon, color, size }) => (
```

Comment:
No caso desse componente, o nome da prop utilizada para dar cor ao icone deveria ser `iconColor`. Ai ao inves de tu mudar o nome da prop aqui pra `color` pra ele usar o propTypes que foi definido em baixo, tu muda o nome na definição do propTypes. 

```suggestion
const Icon = ({ icon, iconColor, size }) => (
```

Reply:
Não quiz mudar propTypes já definidas, mas como esse é caso, mudo sim.